### DISCUSSION TITLE
[Feature] Archive / hide duplicates instead of deleting

### DISCUSSION THREAD

**Discussion by genebean** [immich-app/immich#11269] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have just imported “all the things” to Immic and have nearly 11,000 duplicates, according to the app. I would like to be able to move all the ones that would currently be deleted to an archive or other hidden-from-gallery-by-default place so that they were out of sight, but not gone. This would allow me to hit one button to deal with all 11,000 and know that I could go looking in the duplicates album if I find I’m missing something later. 

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10148

Please use the search like the template requests. It's there for good reason.

**Reply by genebean** [reply]
Totally understand. I did and nothing came up, but I may have just been having crappy service. Sorry I missed it.
